packages/template-tag-codemod/src/index.ts

```diff
@@ -682,9 +693,11 @@ function applyEdits(source: string, edits: { start: number; end: number; replace
 }
 
 export async function run(partialOpts: Options) {
+  // this makes sure that we don't fight with other versions of embroider currently running on the system
+  process.env.EMBROIDER_WORKING_DIRECTORY = 'node_modules/.template-tag-codemod';
```

Comment:
This is too late. Run the code and you'll see that it's not taking effect.

Reply:
yikes you're right, I was driving myself mad trying to figure out why it wasn't working and then I realised it's being ready in **module scope** so there is no way to get it to the "start of execution" 🙈 

I've updated it now and set it just before it is read and confirmed locally that it's working 👍